presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1532,6 +1536,32 @@ public void revokeTablePrivileges(ConnectorSession session, SchemaTableName sche
     }
 
     @Override
+    public List<GrantInfo> listTablePrivileges(ConnectorSession session, String schemaNameOrNull, String grantee)
```

Comment:
Ah, my bad. But listTableColumns takes a `SchemaTablePrefix`, I think that's what I'm remembering -- you probably want to use that so that, if necessary, you can do the requisite filtering to return privileges from one or more tables.